10/01/2013 10:05:50.854 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 83, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 10:53:34.233 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 41, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

10/01/2013 10:54:22.507 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 83, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 10:56:36.453 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 83, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 10:57:04.724 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 83, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 10:57:48.669 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 83, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 11:03:05.5 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 932, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 11:04:22.617 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 932, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 11:05:21.417 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 932, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 11:30:20.880 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 41, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

10/01/2013 11:31:19.438 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 11:34:01.486 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 11:40:49.287 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 706, in run_command
    return rooms_by_name[room_exit.attrs["link"]]
KeyError: u'Upperland'

10/01/2013 11:42:03.347 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 685, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 330, in battle_system
    monster_health = damage(character, int(monster_health))
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 544, in damage
    print character['Name'], 'has done', damage, 'damage'
NameError: global name 'character' is not defined

10/01/2013 11:49:57.670 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 12:37:42.524 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 13:09:39.500 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 931, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 685, in run_command
    battle_system(room, monsters)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 389, in battle_system
    high_score()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 921, in high_score
    choice = raw_input('>>>')
EOFError: EOF when reading a line

10/01/2013 16:53:13.903 Traceback (most recent call last):
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 931, in <module>
    try:
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 810, in run_command
    else:
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 596, in candy_system
    print 'Hp Restored To:', character['player_stats']['health']
ValueError: list.remove(x): x not in list

10/01/2013 16:54:11.20 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 932, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 41, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

10/01/2013 21:16:14.81 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 936, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    print str.startswith(noun)
TypeError: startswith() takes at least 1 argument (0 given)

10/01/2013 21:17:44.531 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 937, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    butt = noun.split[0]
TypeError: 'builtin_function_or_method' object has no attribute '__getitem__'

10/01/2013 21:20:33.404 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 937, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 21:24:35.23 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 939, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 21:26:37.281 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 936, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 21:43:18.916 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    for paths in room.exit.attrs['name']:
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 21:43:58.981 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    for paths in room.exit.attrs['name']:
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 21:44:54.996 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 933, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    print room.exit.attrs['name']
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 21:46:23.938 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 933, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    print room.exit.attrs['name']
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 21:47:03.7 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 933, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 693, in run_command
    print game_map.building.exit.attrs['name']
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 21:47:21.72 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    print game_map.building.exit.attrs['name']
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 21:48:32.105 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    print game_map.building.exit.attrs['dir']
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 21:50:03.421 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    print game_map.building.exit[0].attrs['dir']
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 21:51:30.823 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    for direction in game_map.building.exit[0].attrs['dir']:
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 21:53:06.341 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    for direction in game_map.building[0].exit.attrs['dir']:
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 21:53:10.85 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 21:53:10.223 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 21:57:24.619 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 41, in main
    choice = raw_input("choose a game or type 'N' for a new game\n>")
EOFError: EOF when reading a line

10/01/2013 21:57:31.483 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 935, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    for direction in game_map.building.exit.attrs['dir']:
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 21:59:20.282 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 22:00:17.350 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 22:00:57.804 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 695, in run_command
    print direction.get(noun, None)
AttributeError: 'unicode' object has no attribute 'get'

10/01/2013 22:01:33.243 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 934, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 695, in run_command
    print direction.values()
AttributeError: 'unicode' object has no attribute 'values'

10/01/2013 22:02:34.466 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 937, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 696, in run_command
    butt = str.startswith(verb[0])
TypeError: startswith() takes at least 1 argument (0 given)

10/01/2013 22:03:18.628 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 938, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 22:05:42.784 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 935, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 22:05:56.691 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 935, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 694, in run_command
    for direction in room.exit.attrs['dir']:
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 22:07:02.522 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 936, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 695, in run_command
    for direction in room.exit.attrs['dir']:
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 22:07:54.205 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 938, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 696, in run_command
    for direction in room.exit.attrs['dir']:
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 22:09:02.652 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 938, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 22:09:20.370 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 938, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 696, in run_command
    for direction in game_map.building.exit.attrs['dir']:
AttributeError: 'list' object has no attribute 'exit'

10/01/2013 22:10:02.617 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 935, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

10/01/2013 22:10:18.465 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 935, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 695, in run_command
    print room.exit.attrs['dir']
AttributeError: 'list' object has no attribute 'attrs'

10/01/2013 22:11:21.241 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 936, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 87, in main
    current_room = run_command(current_room, verb, noun)
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 695, in run_command
    for room.exit in room:
TypeError: 'building_q2class' object is not iterable

10/01/2013 22:20:01.905 Traceback (most recent call last):
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 932, in <module>
    main()
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 82, in main
    describe(current_room)
  File "C:\Users\thubbard\Desktop\Project_R_Projects\Text_Adventure\API\Text_of_Mana.py", line 552, in describe
    print room.text[0].value
AttributeError: 'NoneType' object has no attribute 'text'

10/01/2013 22:22:42.148 Traceback (most recent call last):
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 937, in <module>
    main()
  File "C:/Users/thubbard/Desktop/Project_R_Projects/Text_Adventure/API/Text_of_Mana.py", line 84, in main
    command = raw_input(">")
EOFError: EOF when reading a line

